doc/man1/openssl-dgst.pod

```diff
@@ -121,10 +121,16 @@ The actual signature to verify.
 
 =item B<-hmac key>
 
+The L<openssl-mac(1)> command should be preferred to using this command line
+option.
+
```

Comment:
When reading the manual page as a whole, I stumbled over the fact that this remark _precedes_ the description of the option. I would rather prefer it would _follow_ it instead.

_(Applies to all locations)_

Reply:
I vacillated between after and before.

Changed now.